### ISSUE TITLE
Migrate away from JCenter

### ISSUE THREAD

**Issue by alvindizon** [android10/arrow#13]
JCenter will be shut down on May: https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

**Comment by mharlowfod** [comment]
Yes, I am running into this being the final blocker preventing migration away from jcenter in 2 android apps - it would be great to at least have a workaround via jitpack or other if not formal migration to maven or other primary dependency manager.

**Comment by luislugoshockoe** [comment]
Hi, were you able to figure out this dependency issue?
